packages/zql/src/mutate/mutator-registry.ts

```diff
@@ -160,8 +161,8 @@ export function getMutator(
  * Gets a Mutator by its dot-separated name from a MutatorRegistry.
  * Throws if not found.
  */
-export function mustGetMutator(
-  registry: AnyMutatorRegistry,
+export function mustGetMutator<MD extends AnyMutatorDefinitions>(
+  registry: MutatorRegistry<MD>,
```

Comment:
I do not believe this needs to change? why does input need to be strongly typed? We do not return a strongly-typed mutator (input `name` is not constant/keyed).